application/templates/snippets/delete-space.html

```diff
@@ -0,0 +1,5 @@
+<div class="container"> 
+	<div class="email-us">
+		<h5><a href="mailto:example@example.com">Email us to delete your space</a></h5>
```

Comment:
@annawb This should go to anasegroup@gmail.com. Also, maybe have the wording be "Email us to be removed from the database". It's just gentler language.